open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb

```diff
@@ -49,7 +49,9 @@
     "\n",
```

Comment:
<ul><li>Split into multiple chunks of markdown.</li><li>Why are the bolded headings so long, keep the headings short, and focus on the fact that they can be emphasized in other ways.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>The problem hasn't been solved. The aesthetics are terrible.</p>